packages/widget/src/hooks/useCreateEvmWallets.tsx

```diff
@@ -56,18 +57,16 @@ export const useCreateEvmWallets = () => {
 
           try {
             if (isEvmConnected && connector.id !== currentEvmConnector?.id) {
-              await currentConnector?.disconnect();
+              await disconnect(config);
```

Comment:
we need to use the disconnect function from wagmi instead of the disconnect function on the connector itself